tests/test_policy.py

```diff
@@ -14,3 +16,5 @@ def test_policy_checks_glibc():
     assert policy > POLICY_PRIORITY_LOWEST
     policy = versioned_symbols_policy({"some_library.so": {"GLIBC_999"}})
     assert policy == POLICY_PRIORITY_LOWEST
+    policy = versioned_symbols_policy({"some_library.so": {"OPENSSL_1_1_0"}})
+    assert policy == POLICY_PRIORITY_HIGHEST
```

Comment:
@rmcgibbo: So the policy whitelist only applies to GCC*/CXX* symbol versions (see `auditwheel/policy/policy.json` for the full list). Any other symbol version will automatically satisfy the highest policy. If that wasn't the case, then no external dependency symbols (e.g. `OPENSSL_1_1_0`, `KERBEROS_5_MIT`) would satisfy the manylinux1 requirements.